docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,41 +1,124 @@
 === macOS Server
 
-This is only required for building iOS applications.
+This is required for building iOS applications. The following specifications are recommended:
+
+* 4GB RAM
+* 2 Cores
+* macOS 10.12 Sierra
+* 30GB minimum disk space and and additional 20GB per XCode version
 
 ==== macOS Machine Setup
 
-- Set `ansible_sudo_pass` variable for the Ansible job to enable the Installer to work with root permissions. It is required by several installation steps.
+Follow the steps below to enable correct configuration of your macOS server via the installation.
+
+
+====== Create SSH User
+
+Ansible requires SSH access to the macOS server in order to correctly provision the machine to support iOS builds. To enable this, it is required that
+a user with sudo permissions and SSH access is created on the target machine. An example bash script is provided to achieve this. It can be downloaded for
+convenience using the following command: 
+
+`curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/create-osx-user.sh`
 
-.macOS Scripts: https://github.com/aerogear/digger-jenkins/blob/master/admin/create-osx-user.sh[create-osx-user.sh] https://github.com/aerogear/digger-jenkins/blob/master/admin/enable-osx-remote-login.sh[enable-osx-remote-login.sh]
-- Set up a user with sudo permissions and SSH access.
-These scripts will create a user named `jenkins` with SSH access.
-It is highly recommended that you read these scripts and change the `PASSWORD`
-variable in `create-osx-user.sh` to a strong passphrase:
-    
 
---
-  
-. Download the scripts into the target macOS machine:
-+
 [source,bash]
 ----
-curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/create-osx-user.sh
-curl -O https://raw.githubusercontent.com/aerogear/digger-jenkins/master/admin/enable-osx-remote-login.sh
-----
-+
-. Edit the `PASSWORD` for the Jenkins user in `create-osx-user.sh`.
-. Execute `create-osx-user.sh` with the updated password:
-+
-----
-sudo bash create-osx-user.sh
```

Comment:
can we keep this line in. or similar. It's not clear that the file should be executed as root

Reply:
done